packages/components/src/internal/components/picklist/PicklistButton.tsx

```diff
@@ -1,35 +1,49 @@
 import React, { FC, memo } from 'react';
 import { DropdownButton } from 'react-bootstrap';
+import { PermissionTypes } from '@labkey/api';
 
 import { User } from '../base/models/User';
 
 import { QueryModel } from '../../../public/QueryModel/QueryModel';
 
+import { RequiresPermission } from '../base/Permissions';
+
+import { SubMenuItem } from '../menus/SubMenuItem';
+
 import { PicklistCreationMenuItem } from './PicklistCreationMenuItem';
 import { AddToPicklistMenuItem } from './AddToPicklistMenuItem';
 
 interface Props {
     model: QueryModel;
     user: User;
     metricFeatureArea?: string;
+    asSubMenu?: boolean;
 }
 
 export const PicklistButton: FC<Props> = memo(props => {
-    const { model, user, metricFeatureArea } = props;
+    const { model, user, metricFeatureArea, asSubMenu } = props;
 
-    return (
+    const items = (
         <>
-            <DropdownButton title="Picklists" id="samples-picklist-menu">
-                <PicklistCreationMenuItem
-                    itemText="Create Picklist"
-                    selectionKey={model?.id}
-                    selectedQuantity={model?.selections?.size}
-                    key="picklist"
-                    user={user}
-                    metricFeatureArea={metricFeatureArea}
-                />
-                <AddToPicklistMenuItem queryModel={model} user={user} metricFeatureArea={metricFeatureArea} />
-            </DropdownButton>
+            <AddToPicklistMenuItem queryModel={model} user={user} metricFeatureArea={metricFeatureArea} />
+            <PicklistCreationMenuItem
+                selectionKey={model?.id}
+                selectedQuantity={model?.selections?.size}
+                key="picklist"
+                user={user}
+                metricFeatureArea={metricFeatureArea}
+            />
         </>
     );
+
+    return (
+        <RequiresPermission permissionCheck="any" perms={PermissionTypes.ManagePicklists}>
+            {!asSubMenu && (
+                <DropdownButton title="Picklists" id="samples-picklist-menu">
+                    {items}
+                </DropdownButton>
+            )}
+            {asSubMenu && <SubMenuItem text="Picklists">{items}</SubMenuItem>}
```

Comment:
Do we want to display these two menu items inline?  I know this is consistent with the other items, and we can make this change in a later story, but the designs show the items not as submenus.

Reply:
Yes. I confirmed this with Andrew and the plan is to show the items under the "More" button inline as much as possible. The "Derive from Selected" and "Assay" items will still be submenus, but the rest should get inlined with dividers. That said, I don't plan to do that as part of this story because the selenium tests are already passing with this display. We will inline the items as part of the next story.